packages/v4/src/content/design-guidelines/usage-and-behavior/dashboard/dashboard.md

```diff
@@ -0,0 +1,335 @@
+- This information will be best displayed in a list view format.
+
+“How much data will this card contain?”
+- Events happen fast, and in large numbers. This card could contain thousands of events. The user will need to be able to filter this view to tailor it to their specific needs.
+
+“Is the amount of information in the card static or dynamic?”
+- As this is a list that could contain 0 events or could contain thousands, it is indeed dynamic.
+
+Conclusion:
+- This card could contain a massive amount of events. We’ll need to give the card a max height based on the amount of items we think would be reasonable to see at once. Four items at once should be reasonable to set for our max height. The card will then scroll if the user wants to see more events.
+- Since we have a max height of four items and this list view only needs one column, we can estimate that a medium sizing should do.
+
+**When done, your list should look something like this:**
+
+- Details: Small
+- Performance: Large
+- Inventory: Medium
+- Health: Medium
+- Capacity: Medium
+- Top consumers: Large
+- Events: Medium
+
+
+
+### Step 4: Iterate on column structure and card arrangement
+
+Now that you have a prioritized and sized list of cards, use it to create the card layout. To get started, draw your columns in a full size desktop view.
+
+- All columns must be the same width.
+- All columns must have the same sized gutter (space between columns).
+- For a standard desktop size (1200-1400px), try to have the column layout fill the full width of the page.
+- Draw your columns within the context of your project. For our example, we know we have a top masthead, a top header information area, a left hand navigation, and a canvas for the dashboard.
+
+You can use any common amount of columns for this step, but three and four column layouts are the most common. If you use Sketch, you can find templates for these layouts in the [PatternFly template](https://www.sketch.com/s/729c2eee-e8b6-4fcd-8a79-f6faa8c30f89). For the purpose of this tutorial, we will work with a standard four column layout.
+
+<img src="./img/dashboard-4-column.png" alt="four column dashboard layout" width="1680"/>
+
+Four column layout with 16 px gutters. There are also 16px margins around the dashboard.
+
+Next, add your cards in their appropriate sizes without any content. Don’t worry about the arrangement yet, just draw all the cards at your estimated sizes.
+
+- There should be no real content in your cards yet, only each card’s name. At this point, the only thing we’re concerned about is getting a general size for each card.
+- Again, these are guesstimates. We’re not worried about the exact height of each card. Rather, we’reusing our column structure as a guide.
+
+
+After placing our prioritized and sized card list on the four-column grid, the result should look like this example:
+
+<img src="./img/dashboard-first-cut.png" alt="dashboard layout populated with placeholder cards" width="1680"/>
```

Comment:
This is nitpicky, but if you have access to the image, could you change the font to red hat font instead of overpass?